iotilecore/iotile/core/hw/transport/adapter/abstract.py

```diff
@@ -419,18 +506,19 @@ def remove_monitor(self, handle):
             RPCErrorCode: The RPC implementation wishes to fail with a
                 non-zero status code.
             RPCInvalidIDError: The rpc_id is too large to fit in 16-bits.
+            TileBusSerror: The tile was busy and could not respond to the RPC.
```

Comment:
Is there a typo here? That "S" sticks out

Reply:
yeah, `TileBusyError`